core/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java

```diff
@@ -608,8 +619,14 @@ boolean isPreserveAliases() {
                     result = new HashSet<>(expressions.subList(0, i));
                 }
                 if (!Regex.isSimpleMatchPattern(expression)) {
-                    if (!unavailableIgnoredOrExists(options, metaData, expression)) {
-                        throw infe(expression);
+                    //TODO why does wildcard resolver throw exceptions regarding non wildcarded expressions? This should not be done here.
```

Comment:
I tried to remove this logic but tests start to fail, I will dig deeper another time.